plugins/foreman_remote_execution/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Remote Execution Documentation
-versions: [1.7, 1.3, 0.3, 0.2, 0.1, 0.0]
+versions: [10.0.3, 1.7, 1.3, 0.3, 0.2, 0.1, 0.0]
```

Comment:
Not sure why this renders so poorly, but this should also be `10.0`.
```suggestion
versions: [10.0, 1.7, 1.3, 0.3, 0.2, 0.1, 0.0]
```

Reply:
Applied.